nbbuild/travis/scripting.sh

```diff
@@ -28,24 +28,15 @@ if [ -z "$GRAALVM" ]; then
   GRAALVM=`pwd`/$BASE
 fi
 
-# test on regular VM
-
-ant -f platform/api.scripting/build.xml test
```

Comment:
Should this really be removed?

Reply:
Tests on regular VM are executed in other Travis jobs like this: https://github.com/apache/netbeans/blob/master/.travis.yml#L362